### DISCUSSION TITLE
files and directories not belonging to user in .local/share/containers/ after running  rootless podman

### DISCUSSION THREAD

**Discussion by mrdiago** [containers/podman#18802] [Q&A] [answered]
using arch linux with podman 4.5.1, linux 6.3.5 
when I pull the image all files an directories in .local/share/containers belong to user running podman pull
when I run the image
some files or directories are created and not belong to user

podman run command

`/usr/bin/podman run \
--cidfile=%t/%n.ctr-id \
--cgroups=no-conmon \
--rm \
--sdnotify=conmon \
--user 1000:1000 \
--detach \
--replace \
--label io.containers.autoupdate=registry \
--uidmap 1000:0:1 \
--uidmap 0:1:1000 \
--uidmap 1001:1001:64536 \
--network=slirp4netns:allow_host_loopback=true,cidr=10.0.2.0/24,port_handler=slirp4netns \
--dns=1.1.1.1 \
--name=navidrome \
--cap-add=NET_ADMIN \
--env PUID=1000 \
--env PGID=1000 \
--env TZ=Europe/Paris \
--publish 4533:4533 \
--volume /home/navidrome/data:/data \
--volume /media/opt2/mp3/fromflac:/music:ro docker.io/deluan/navidrome:latest`

extract of find /home/navidrome ! -jser navidrome 
e0d1f/diff/usr/share/X11/locale/nokhchi-1
 11012050      4 -rw-r--r--   1 689824   689824        341 févr.  4 22:02 /home/navidrome/.local/share/containers/storage/overlay/be8d520ade174d418abac328f66820597fbd8f2c172cc502d82bf01ad3
0e0d1f/diff/usr/share/X11/locale/nokhchi-1/XI18N_OBJS
 11012051      4 -rw-r--r--   1 689824   689824        908 févr.  4 22:02 /home/navidrome/.local/share/containers/storage/overlay/be8d520ade174d418abac328f66820597fbd8f2c172cc502d82bf01ad3
0e0d1f/diff/usr/share/X11/locale/nokhchi-1/XLC_LOCALE
 11012049      0 -rw-r--r--   1 689824   689824          0 févr.  4 22:02 /home/navidrome/.local/share/containers/storage/overlay/be8d520ade174d418abac328f66820597fbd8f2c172cc502d82bf01ad3
0e0d1f/diff/usr/share/X11/locale/nokhchi-1/Compose
 11012107      4 drwxr-xr-x   2 689824   689824       4096 févr. 14 02:30 /home/navidrome/.local/share/containers/storage/overlay/be8d520ade174d418abac328f66820597fbd8f2c172cc502d82bf01ad3
0e0d1f/diff/usr/share/X11/locale/vi_VN.tcvn
 11012115      4 -rw-r--r--   1 689824   689824        342 févr.  4 22:02 /home/navidrome/.local/share/containers/storage/overlay/be8d520ade174d418abac328f66820597fbd8f2c172cc502d82bf01ad3
0e0d1f/diff/usr/share/X11/locale/vi_VN.tcvn/XI18N_OBJS
 11012119      4 -rw-r--r--   1 689824   689824        907 févr.  4 22:02 /home/navidrome/.local/share/containers/storage/overlay/be8d520ade174d418abac328f66820597fbd8f2c172cc502d82bf01ad3
0e0d1f/diff/usr/share/X11/locale/vi_VN.tcvn/XLC_LOCALE

**Comment by Luap99** [comment, marked as answer]
This is expected, podman uses ids from /etc/subuid and /etc/subgid. For example see https://www.redhat.com/sysadmin/rootless-podman-user-namespace-modes for an explanation on how it works.
You can use `podman unshare` to access these files.

**Reply by mrdiago** [reply]
thanks you for your replies 
the uidmap allow to map one user inside to one user outside only, when several users manipules files inside the containers there is no way to make these files readable and writable by user outside the container ( for backup purpose )

but as you indicate : we can use podman unshare to launch a backup command
